### ISSUE TITLE
Allow opening files from explorer/cli in an existing Nvy instance

### ISSUE THREAD

**Issue by nickbrooking** [RMichelsen/Nvy#135]
It is possible to drag a file from explorer on to Nvy and it will open the file in that instance, which is great. However it would be nice to allow the same behavior just from double clicking a file in explorer or via the command line.

**Comment by jpt13653903** [comment]
I would like to vote against this idea...

Or, rather, make it configurable if possible.

If I want it to open in the same instance, I use the built-in file explorer, or Oil, or telescope, or similar in the instance that I'm working in.

If I double-click in Windows Explorer it is because I specifically want a new instance so that I can put it on the other screen.

But.  This said...

It would be really nifty if there can be a command-line option like `--existing-instance` so that it opens in an existing instance when I select the file in something like Quartus (an FPGA firmware IDE).

In that case, it would need quite a lot of logic behind the scenes...  It needs to search the existing instances for in case the file is already open and then  choose it, otherwise choose a random instance.

**Comment by nickbrooking** [comment]
I agree that this behavior could be configurable, but I will say that most other modern editors (VSCode, Notepad++, even just normal Notepad) behave this way by default.

**Comment by jpt13653903** [comment]
Everybody else is jumping off cliffs, so you also have to ;-)

In all seriousness though -- I don't like the behaviour of "other modern editors", which is why I don't use them.

I'm going to remove myself from this discussion...  I've said my piece.

**Comment by Tordan** [comment]
I forked this repository specifically to add this feature . You can check it out [here](https://github.com/Tordan/Nvy/tree/add-reuse-window-option). Now you can use it like `Nvy.exe --reuse-window d:\foo.txt` and this file will be opened in the existing window.
